DLplatform/worker.py

```diff
@@ -236,6 +236,7 @@ def onCommunicatorMessageReceived(self,  routing_key, exchange, body):
             param = message['param']
             flags = message['flags']
             self._learner.setModel(param, flags)
+            self._communicator.learningLogger.logIntermediateModel(param) #logs the intermediate model
```

Comment:
I guess now you have it in two places, where you log the model?

Reply:
You can see  in the new pull request that 'after' models are logged in DLplatform/worker.py : onCommunicationMessageReceived() function and 'before' models are logged in the setModel() function in DLplatform/learning/learner.py

Reply:
okey, I see